src/types/email_mode_data.ts

```diff
@@ -1,30 +1,4 @@
-// TODO (#42): remove these types when migrating away from middleware pattern
```

Comment:
~~can we camelCase the file name instead? :P~~
edit: nvm we wrote ourselves into a corner with the previous names.....